hathor/p2p/protocol.py

```diff
@@ -82,7 +82,7 @@ class WarningFlags(str, Enum):
     connection_string: Optional[str]
     expected_peer_id: Optional[str]
     warning_flags: Set[str]
-    connected: bool
+    is_connected: bool
```

Comment:
We have discussed this parameter before and it was renamed to `aborting` (flipping the boolean value) on one of the sync-v2 branches. The main difference being that `aborting` starts with `False` which is not the inverse of `connected`, which also starts with `False`, so this property is only changed when we decide to close a connection, and not when it is connected. Which from what I remember from what we discussed it is best to leave the state of the connection to twisted.

Reply:
I had to rename it because `connected` is used by the parent class and we do not have full control over it. But I agree that `aborting` is a better way to deal with it.

Reply:
These are the changes: https://github.com/HathorNetwork/hathor-core/pull/217/files#diff-e0be12551dd55b1003c020ef234d6a44a5126754c055b1a96fcdb3900b5be622 if you want to move them to this PR it will avoid renaming it twice in a row.